### PR TITLE
Mailchimp - Custom merge fields

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added custom user traits

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/destinations/marketing/mailchimp.mdx [hunk 1]
```diff
@@ -1,99 +1,116 @@
...
 
-Let us consider a sample `identify` call in RudderStack:
+A sample `identify` call is shown below:
 
 ```javascript
-rudderanalytics.identify("user123", {
-  firstName: "John",
-  lastName: "Doe",
-  email: "namesurname@domain.com"
+rudderanalytics.identify("1hKOmRA4el9Zt1WSfVJIVo4GRlm", {
```
Comment (shrouti1507) [inline docs/destinations/marketing/mailchimp.mdx:61]: can we please make it like "user123" in this one and all the following examples? It is more intuitive in that way.

File: docs/destinations/marketing/mailchimp.mdx [hunk 2]
```diff
@@ -105,19 +122,17 @@ rudderanalytics.identify(
...
-RudderStack gives you the option of manually updating the subscription status of a user. Simply pass on an additional parameter to Mailchimp called `subscriptionStatus`.
+RudderStack provides you with the option to manually update the subscription status of a user by passing the `subscriptionStatus` parameter to Mailchimp.
 
-An example of how to do this is as shown in the code snippet below:
+An example is shown below:
 
 ```javascript
 rudderanalytics.identify(
-  "userId123",
+  "1hKOmRA4el9Zt1WSfVJIVo4GRlm",
```
Comment (shrouti1507) [inline docs/destinations/marketing/mailchimp.mdx:133]: here too

**Review by shrouti1507 — COMMENTED** [review]
Rest looks good

**Review by rohithkaza — APPROVED** [review]

File: docs/destinations/marketing/mailchimp.mdx (hunk 1 above)
Comment (shrouti1507) [reply docs/destinations/marketing/mailchimp.mdx:61]: got it

**Review by shrouti1507 — APPROVED** [review]

**Comment by ishachopra0401** [comment]
Closing this PR and opening a new one to resolve conflicts

**Pull request closed without merging**